src/web/App.mjs

```diff
@@ -824,6 +856,67 @@ class App {
         this.loadURIParams();
     }
 
+
+    /**
+     * Update element visibility
+     *
+     * @param {HTMLElement} elm
+     * @param {boolean} isVisible
+     *
+     */
+    updateVisibility(elm, isVisible) {
+        return isVisible ? elm.classList.remove("hidden") : elm.classList.add("hidden");
+    }
+
+    /**
+     * Set desktop UI ( on init and on window resize events )
+     *
+     * @param {boolean} minimise
+     */
+    setDesktopUI(minimise) {
+        $("[data-toggle=tooltip]").tooltip("enable");
+        this.setCompileMessage();
+        this.setDesktopSplitter(minimise);
+        this.adjustComponentSizes();
+        this.populateOperationsList();
+        this.manager.recipe.clearAllSelectedClasses();
+    }
+
+    /**
+     *  Set mobile UI ( on init and on window resize events )
+     */
+    setMobileUI()  {
+        $("[data-toggle=tooltip]").tooltip("disable");
```

Comment:
Use `document.querySelector('[data-toggle=tooltip]')` to remove reliance on jQuery

Reply:
No can do, unless you have other ideas, I'm leaving it as is